data/controller_i18n_links.yaml

```diff
@@ -0,0 +1,509 @@
+# Translation links for Controller
+# Start Korean translations
+tower_ko:
```

Comment:
this is a total nit but if these are all controller pages, why do we use things like tower_ko etc throughout the new docsite? Maybe that's an easyfix issue we can create for someone else to come in and clean that up?

Reply:
yeah we could create an easyfix issue for that. it was inherited from the names of the HTML pages on the docsite (e.g. `automation-tower-chinese-translations.html`) so I just made the key match the page. we can totally change that though.